lib/raidenclient/RaidenClient.ts

```diff
@@ -511,6 +511,10 @@ class RaidenClient extends SwapClient {
     return body.channel_address;
   }
 
+  public listChannels = (_activeOnly: boolean): Promise<Channel[]> => {
```

Comment:
FWIW, Raiden does have a method to [list channels](https://raiden-network.readthedocs.io/en/stable/rest_api.html#querying-information-about-channels-and-tokens), however I'm not seeing how to determine whether the partner for a channel is online & active or not, so we couldn't calculate inactive balance as far as I can see anyhow.